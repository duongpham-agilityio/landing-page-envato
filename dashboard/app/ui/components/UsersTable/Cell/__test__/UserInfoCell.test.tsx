import { render } from '@testing-library/react';
import { UserInfoCell } from '..';
import { Table, Tr } from '@chakra-ui/react';

describe('TotalCard component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Table>
        <Tr>
          <UserInfoCell name="test" imageURL="url" email="test@gmail.com" />
        </Tr>
      </Table>,
    );

    expect(container).toMatchSnapshot();
  });
});
