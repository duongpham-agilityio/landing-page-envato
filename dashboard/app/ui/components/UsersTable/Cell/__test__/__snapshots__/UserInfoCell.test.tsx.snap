// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalCard component renders correctly 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <tr
      class="css-0"
    >
      <td
        class="css-mzqydp"
      >
        <div
          class="css-1h2sv0j"
        >
          <img
            alt="Image of test"
            class="chakra-image css-1rqxmuh"
            src="url"
          />
          <div
            class="css-1rr4qq7"
          >
            <h4
              class="chakra-text css-197ngd6"
            >
              test
            </h4>
            <p
              class="chakra-text css-vxhtcb"
            >
              <span
                class="chakra-text css-1rgac5r"
              />
            </p>
          </div>
        </div>
      </td>
    </tr>
  </table>
</div>
`;
